addons/website_mail_channel/i18n/website_mail_channel.pot

```diff
@@ -1,397 +0,0 @@
-# Translation of Odoo Server.
```

Comment:
Don't we typically keep po/pot files in place on module removal to avoid noise in the PR, and let the bot clean them up at translation export before the release?

Reply:
Did not know this, thanks :)

Reply:
> Don't we typically keep po/pot files in place on module removal to avoid noise in the PR, and let the bot clean them up at translation export before the release?

Not when removing a module, as anyway everything would be generated again in other module POT files. Here we somehow 'keep' the module but with an entirely different content (as most of it is moved to mail_group) so I suppose both are fine. Maybe ask Martin to be sure.